app/decorators/user_decorator.rb

```diff
@@ -40,7 +40,7 @@ def masked_two_factor_phone_number
   end
 
   def identity_not_verified?
-    user.active_profile ? false : true
+    !user.verified?
```

Comment:
I don't disagree with you.

I've left it there because I have heard from other Ruby developers that they prefer this syntax:

```ruby

if identity_not_verified?  # preferred
if !user.verified?  # less good
unless user.verified?   # least good
```

The reason is that when you combine a positive assertion and a negative assertion in a single statement, the logic can be hard to follow. Currently the method is used like this:

```ruby
  def identity_needs_verification?
    loa2_or_above? && identity_not_verified?
  end
```

which, if we got rid of that method, would look like:

```ruby
  def identity_needs_verification?
    loa2_or_above? && !user.verified?
  end
```

which is (arguably) harder to grok.